source/simulator/assemblers/stokes.cc

```diff
@@ -269,6 +269,12 @@ namespace aspect
       const MaterialModel::ElasticOutputs<dim>
       *elastic_outputs = scratch.material_model_outputs.template get_additional_output<MaterialModel::ElasticOutputs<dim> >();
 
+      const MaterialModel::PrescribedCompressionOutputs<dim>
+      *prescribed_compression =
+        (this->get_parameters().enable_prescribed_compression)
+        ? scratch.material_model_outputs.template get_additional_output<MaterialModel::PrescribedCompressionOutputs<dim> >()
+        : nullptr;
```

Comment:
Just don't use a parameter and instead test whether `get_additional_output()` returns anything that's non-null to determine whether or not to assemble anything.

Reply:
This makes the logic below (``const bool update_strain_rate = ...``) complicated as I would need to create a fake material model.